ui/src/app/components/gke-app-configuration-panels/create-gke-app-panel.tsx

```diff
@@ -59,9 +59,9 @@ export const CreateGKEAppPanel = ({
   app,
   disk,
   onClickDeleteUnattachedPersistentDisk,
-  IntroText = () => null,
-  PostCost = () => null,
-  PostCompute = () => null,
+  introTextOverride,
```

Comment:
Instead of using "override" terminology and adding additional logic, you could set the default value on this line. This is where I would look for a default value for a function parameter marked as optional.

Reply:
makes sense.  updated to `introText`